deeptrack/optics.py

```diff
@@ -1,20 +1,172 @@
-""" Features for optically imaging of samples
-
-Contains features which performs physical simulations of optical devices to
-create camera images of samples.
-
-Classes
--------
-Microscope
-    Image a sample using an optical system.
-Optics
-    Abstract base optics class.
-Fluorescence
-    Optical device for fluorescenct imaging.
-Brightfield
-    Images coherently illuminated samples.
 """
+Features for Optical Imaging of Samples.
```

Comment:
Questo andrebbe:

"""Features for optical imaging of samples.

(single line, not camel case)